manifests/ruby.yml

```diff
@@ -399,12 +395,13 @@ tests/:
       TestTracerFlareV1: missing_feature
   remote_config/:
     test_remote_configuration.py:
+      Test_Agent: v1.13.0
```

Comment:
I'm not sure about that, I have not seen it in any other manifest but there are decorators above the tests, which I believe should be included in manifests instead